### ISSUE TITLE
[fvt]2.12.1 xcatprobe image could not support image  installed on computenode  if no releated xcatinfo   on managemenent node

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1412]
env:ubuntu
build:root@c910f03c05k05:/opt/xcat/bin# lsxcatd -v
Version 2.12.1 (git commit 5431a2ab9ccd42f5f76540ec0868429dd6199056, built Sun Jun 26 06:47:12 EDT 2016)
How to reproduce:
```
1.imgexport rhels7.2-ppc64le-netboot-compute
2.imgimport rhels7.2-ppc64le-netboot-compute
3. nodeset c910f03c05k09 osimage= rhels7.2-ppc64le-netboot-compute
4.rpower c910f03c05k09 boot
Connection to c910f03c05k02 closed.
root@c910f03c05k05:/opt/xcat/bin# ./xcatprobe image -c
Smartmatch is experimental at ./xcatprobe line 64.
Smartmatch is experimental at ./xcatprobe line 173.
       c910f03c05k02 is diskless
       c910f03c05k09 is diskless
[ OK ] Pinging c910f03c05k02
[ OK ] Pinging c910f03c05k09
       ---- Gathering information from node c910f03c05k02 ----
awk: fatal: cannot open file `/install/netboot/rhels7.2/ppc64le/compute/rootimg/opt/xcat/xcatinfo' for reading (No such file or directory)
       ---- Gathering information from node c910f03c05k09 ----
[FAIL] Not all compute nodes are installed with the same os image.
c910f03c05k02   rhels7.2-ppc64le-netboot-compute : fd7eeb7e-f243-4d4c-9884-d3409c8f8668
c910f03c05k09   ubuntu14.04.4-ppc64el-netboot-compute : 81cd45bf-e463-42ee-bc21-e86e21ddeec4
```

**Comment by junxiawang** [comment]
if no xcatinfo on management node xcatprobe image  could not work
```
if (length($rootimagedir) > 0) {
            $defined_UUID = `awk -F"'" '/IMAGEUUID/ {print \$2}' $rootimagedir/rootimg/opt/xcat/xcatinfo`;
```
please have a check if no xcatinfo in related image ,for xcat 2.12 and before no xcatinfo in releated image

**Comment by gurevichmark** [comment]
Fixed under #1403
